### PR TITLE
{geo}[intel/2019b] WRF v4.1.3 + WPS v4.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1884~~ (WRF easyblock fix) + ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1886~~ (WPS easyblock update) + https://github.com/easybuilders/easybuild-easyblocks/pull/1895 (WPS easyblock fix) + ~~#9496~~ (tcsh) + ~~#9494~~ (netCDF) + ~~#9497~~ (netCDF-Fortran)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e2d301587ee95f85a102405743ec1d9e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2450.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6d42b7b82059ab613c240e2bd9e77924 for a full test report.

**Comment by boegel** [comment]
required patch is available in #7085, I'll try and get that one merged first...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3133.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/3be36a1a4dbca9650341feb71addcc7e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2499.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1bbca35b783324e841bc16bb144acf3d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
generoso-2 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/faa4ed13b1b4a33638368f070bb0cb3e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3139.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/17d2e02dbab517ba4aff049d55de5262 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2598.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3234fdfe83d1e25fa018bf6a14e92916 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
generoso-4 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/250ec6427851a773e91919b3abe94554 for a full test report.

File: easybuild/easyconfigs/w/WPS/WPS-4.1_netCDF-Fortran_seperate_path.patch [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+-  if [ -f "$NETCDF/lib/libnetcdff.a" ] ; then
+-    NETCDFF="-lnetcdff"
++  USENETCDFF=" "
++  # for netCDF 4.2 and greater, the Fortran library is a seperate install
++  if [ -z "$NETCDFF" ]
++  then
++    NETCDFF=$NETCDF
++  fi
++  if [ -f "$NETCDFF/lib/libnetcdff.a" ] ; then
++    USENETCDFF="-L$NETCDFF/lib!-lnetcdff"
```
Comment (smoors) [inline easybuild/easyconfigs/w/WPS/WPS-4.1_netCDF-Fortran_seperate_path.patch]: @boegel I guess this should be:
```
USENETCDFF="-L$NETCDFF/lib -lnetcdff"
```

Comment (boegel) [reply easybuild/easyconfigs/w/WPS/WPS-4.1_netCDF-Fortran_seperate_path.patch]: No, there's a reason for this I think... The `!` is replaced with a space later somewhere.

It's also like this in the original patch `WPS_netCDF-Fortran_seperate_path.patch`

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 8 out of 8 (3 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/3180a706e4ba618d8de85b2514fabf41 for a full test report.

**Comment by boegel** [comment]
@smoors: Good to go?

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**